app/views/root/designprinciples.html.erb

```diff
@@ -180,9 +180,10 @@
             think about those people from the start.
            </p>
            <ul>
+             <li><a href="https://accessibility.blog.gov.uk/2016/05/16/what-we-mean-when-we-talk-about-accessibility-2/"></a>What we mean when we talk about accessibility, by&nbsp;Alistair&nbsp;Duggin</li>
+             <li><a href="https://accessibility.blog.gov.uk/2016/05/16/consider-the-range-of-people-that-will-use-your-product-or-service/">Consider the range of people that will use your product or service, by by&nbsp;Alistair&nbsp;Duggin</a></li>
```

Comment:
Looks like a copy and paste error - there's a double, "by by" Alistair. Once fixed I'm happy to merge.